### ISSUE TITLE
[Minor issue] Consolidate items into as few stacks as possible

### ISSUE THREAD

**Issue by Midnighit** [bigmanshawn/ConanEvolved_Merchants#21]
When dragging items out of a merchant they are still in the same quantities they were when sold to the merchant/dragged to them. So if I drag over 3 items and later another 2 of the same type to the merchant I'll get the same 3 and 2 again when dragging them back to me (as opposed to the whole 5 items). I suggest to always consolidated all items to as few stacks as possible.

**Comment by bigmanshawn** [comment]
Yeah this is a known issue but has been pretty down there on the list of things to work on.  Thanks for bringing it back up.
